mods/ts/rules/defaults.yaml

```diff
@@ -668,6 +668,31 @@
 		Terrain: Tree
 	Tooltip:
 		Name: Tree
+	Health:
+		HP: 500
+	Armor:
+		Type: Wood
+	Targetable:
+		TargetTypes: Trees
+	WithDamageOverlay@SmallBurn:
+		DamageType: Incendiary
+		Image: burn-s
+		MinimumDamageState: Light
+		MaximumDamageState: Dead
+		Palette: effect
+	WithDamageOverlay@MediumBurn:
+		DamageType: Incendiary
+		Image: burn-m
+		MinimumDamageState: Medium
+		MaximumDamageState: Dead
```

Comment:
Oh, I thought they were overlayed. Then I probably chose the wrong SHPs as well.